build.gradle

```diff
@@ -139,7 +139,7 @@ wrapper.gradleVersion = '2.10'
 dependencies {
     compile "com.fasterxml.jackson.core:jackson-databind:2.10.0"
 
-    compile group: 'io.rest-assured', name: 'rest-assured', version: '3.1.0'
+    compile group: 'io.rest-assured', name: 'rest-assured', version: '4.4.0'
```

Comment:
nice, thanks.

Reply:
but we need to make sure major rest assured upgrade doesn't bring breaking changes to currently adopting repos. Please raise a PR against CCD Data Store and CCD Def store develop branches just in case. I'm not expecting a breaking aspect, though.

Reply:
The projects data store and definition store already have rest assured version 4.3.0.
Just to be double sure, I have raised 2 PR's
1. https://github.com/hmcts/ccd-data-store-api/pull/1515/files
2. https://github.com/hmcts/ccd-definition-store-api/pull/1015

Both are green with the latest BEFTA jar, so we should be good.